docs/tutorials/create-a-resource-node.md

```diff
@@ -0,0 +1,106 @@
+## Overview
+
+This tutorial will walk you through adding a piece of digital content (an image and some metadata) into an Islandora repository.
+At the end, you will have metadata in a Drupal node, a file stored in Fedora, and derivatives automatically created.
+
+!!! Tip "Video version available"
+    The material in this tutorial is presented in our video, [Adding Content](https://youtu.be/G52is7iFkG4).
+
+## Tools
+
+For this tutorial you will need an instance of [Islandora Defaults](https://github.com/Islandora/islandora_defaults). You can use the online sandbox or one of the [sandbox VMs](https://islandora.ca/try).
+
+[Online Sandbox](http://future.islandora.ca)
+
+* Username: Test
+* Password: islandora
+
+## Step 1: Create an Islandora Node
+
+To add content to Islandora, start by creating a node. When logged in to Drupal as an administrator, navigate with the top menu bar to **Manage** >> **Content** >> **Add content** or click on **Add content** under the _Tools_ menu.
+
+![Click on Add Content](../assets/create_node_add_content.png)
+
+Next, select a content type that is configured with Islandora behaviours. In Islandora Defaults, there is Islandora content type. It is called **Repository Item**.
+
+![Click on Repository Item](../assets/create_node_select_repo_item.png)
+
+You are presented with a form showing a large number of metadata fields. Fill out the ones that are applicable to you.
+
+!!! Error "Keep it simple"
+    For this tutorial, do not attempt to fill out any fields with an autocomplete symbol - a small circle at the right-hand-side of the text box.
+
+<!-- We should really include creating/adding a name here or in another tutorial!! It's a source of immediate confusion and frustration! -->
+![Avoid autocomplete fields](../assets/create_node_warning_autocomplete.png)
+<!-- Can we get this image inside of the callout box? -->
```

Comment:
Out of curiosity, what happens when you try?

Reply:
Ooh. Tried it, and doing it like this works: 

```
!!! Error "Keep it simple"
    For this tutorial, do not attempt to fill out any fields with an autocomplete symbol - a small circle at the right-hand-side of the text box. ![Avoid autocomplete fields](../assets/create_node_warning_autocomplete.png)
```

Reply:
![image](https://user-images.githubusercontent.com/2371345/99718800-d48b9f80-2a81-11eb-8e7c-487df1491bc1.png)